modules/core/src/main/java/org/apache/ignite/spi/systemview/view/ServiceView.java

```diff
@@ -98,4 +99,14 @@ public boolean staticallyConfigured() {
     public UUID originNodeId() {
         return serviceInfo.originNodeId();
     }
+
+    /**
+     * Gets service deployment topology snapshot. Service topology snapshot is represented
```

Comment:
Other descriptions of the properties  are short int this class. Maybe just '@return Map of number of service instances per node ID.' and '@see ServiceDescriptor'? WDYT?

Reply:
Done